src/OrbitQt/HistogramWidget.cpp

```diff
@@ -321,7 +321,7 @@ static void DrawSelection(QPainter& painter, int start_x, int end_x,
 [[nodiscard]] static uint64_t LocationToValue(int pos_x, int width, uint64_t min_value,
                                               uint64_t max_value) {
   if (pos_x <= kLeftMargin) return min_value;
-  if (pos_x > width - kRightMargin) return max_value + 1;
+  if (pos_x > width - kRightMargin) return max_value;
```

Comment:
Why you are modifying this? Was this a bug? Can you explain it in the body of the PR?

Reply:
Yeah, it's a but that became noticeable with this change.